requirements.txt

```diff
@@ -1,18 +1,18 @@
-django==3.2.25
-django-tagging==0.5.0
-django-reversion==4.0.0
-django-bootstrap3==21.2
-django-summernote==0.8.11.6
-django-modeltranslation==0.18.2
-django-registration==3.1.2
-django-contrib-comments==1.9.2
-django-queryset-csv==1.1.0
-django-debug-toolbar==3.2.2
-django-guardian==2.2.0
-django-notifications-hq==1.6.0
-dj-database-url==0.5.0
-boto3==1.22.8
-django-storages==1.12.1
+django~=4.2.13
```

Comment:
```suggestion
django ~= 4.2
```

Reply:
It fails OSV if I do that.

Reply:
right, because `requirements.txt` is not a lockfile and so it assumes the lowest version which has vulnerabilities - you should use an exact version instead:

```suggestion
django == 4.2.13
```

Reply:
Ok.

Reply:
Done.